src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -735,15 +735,20 @@ public class Views : RevitDropDownBase
         protected override SelectionState PopulateItemsCore(string currentSelection)
         {
             Items.Clear();
-            //find all views in the project
-            var fec = new FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument);
-            var views = fec.OfClass(typeof(View)).ToElements();
+            // find all views in the project
+            // exclude <RevisionSchedule>
+            var views = new FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument)
+                .OfClass(typeof(View))
+                .Where(x => !x.Name.Contains('<'))
+                .ToList();
             
-            //there must always be at least 1 view in a Revit document, so we can exclude the empty list check
+            // there must always be at least 1 view in a Revit document, so we can exclude the empty list check
             foreach (var v in views)
             {
                 Items.Add(new DynamoDropDownItem(v.Name, v));
             }
+            Items = Items.OrderBy(x => x.Name).ToObservableCollection();
```

Comment:
@sm6srw what do you think of this sorting ? Is this going to cause the issue we talked about the other day?